src/components/Tokens/TokenDetails/index.tsx

```diff
@@ -148,21 +181,21 @@ export default function TokenDetails({
             </TokenInfoContainer>
             <ChartSection priceQueryReference={priceQueryReference} refetchTokenPrices={refetchTokenPrices} />
             <StatsSection
-              TVL={tokenQueryData.market?.totalValueLocked?.value}
-              volume24H={tokenQueryData.market?.volume24H?.value}
-              priceHigh52W={tokenQueryData.market?.priceHigh52W?.value}
-              priceLow52W={tokenQueryData.market?.priceLow52W?.value}
+              TVL={tokenQueryData?.market?.totalValueLocked?.value}
+              volume24H={tokenQueryData?.market?.volume24H?.value}
+              priceHigh52W={tokenQueryData?.market?.priceHigh52W?.value}
+              priceLow52W={tokenQueryData?.market?.priceLow52W?.value}
             />
-            {!isNative && (
+            {!isNative && address && (
```

Comment:
dangerous to use truthy values in compound-boolean renders--should wrap this or make a standalone descriptive variable